src/runtime/composables/useFormGroup.ts

```diff
@@ -62,7 +63,8 @@ export const useFormGroup = (inputProps?: InputProps, config?: any) => {
     name: computed(() => inputProps?.name ?? formGroup?.name.value),
     size: computed(() => {
       const formGroupSize = config.size[formGroup?.size.value as string] ? formGroup?.size.value : null
-      return inputProps?.size ?? formGroupSize ?? formGroupConfig?.default?.size
+      const defaultSize = config.default?.size ?? formGroupConfig?.default?.size
```

Comment:
I guess `?? formGroupConfig?.default?.size` is not necessary as `config.default?.size` will always be defined.